src/cyw43_ll.c

```diff
@@ -1849,7 +1850,7 @@ int cyw43_ll_wifi_on(cyw43_ll_t *self_in, uint32_t country) {
     if (country == CYW43_COUNTRY_WORLDWIDE) {
         // For the 1DX CLM, we need to use revision 17 for the worldwide
         // country. This increases throughput by about 25% on PYBD.
-        country |= (17 << 16)
+        country |= (17 << 16);
```

Comment:
please squash this fix into the other commit (also the include of cyw43_country.h)